mimic/model/lstm_model.py

```diff
@@ -32,15 +32,16 @@ def learn(self, text):
         """Use input text to train the LSTM model."""
         # Clean & verify text
         logging.info('Cleaning and verifying text')
-
+        text = text[:150000]
         clean_txt = utils.clean_text(text)
         txt_len = len(clean_txt)
-        utils.verify_text(clean_txt)
+        # utils.verify_text(clean_txt)
```

Comment:
Why was this commented out?

Reply:
This was checking word count to be > 20000 which wasn't satified for smaller corpora, but we can add it back in now